### ISSUE TITLE
Change mouse wheel sensitivity runtime on Windows

### ISSUE THREAD

**Issue by wenyue** [flutter/flutter#149086]
### Use case

Flutter reads the Windows Mouse system configuration at startup, but can not be modified at run time. We should be able to configure different mouse wheel sensitivity for each flutter program, at runtime.

### Proposal

Add a function to modify the sensitivity  of the mouse wheel.

**Comment by darshankawar** [comment]
@wenyue 
Can you check if there's any community package that already offers this feature ? Maybe https://pub.dev/packages/dyn_mouse_scroll

Also, can you check if the sensitivity can be adjusted via mouse settings itself that helps in your case ?

**Comment by wenyue** [comment]
No, I hope flutter can modify it globally instead of modifying each widget individually.

**Comment by darshankawar** [comment]
@wenyue Isn't [this](https://github.com/flutter/flutter/issues/107248) issue similar to your case ? Because this comment seems to also say same as current behavior you mentioned ? 

`After https://github.com/flutter/engine/pull/32094, Flutter apps read this scroll speed setting once at startup. However, changing the scroll speed setting doesn't update Flutter apps that are already running.`
